task/gitleaks/0.1/gitleaks.yaml

```diff
@@ -0,0 +1,79 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: gitleaks
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.19.0"
+    tekton.dev/tags: Security, Secrets
+    tekton.dev/displayName: "Gitleaks"
+spec:
+  description: >-
+    This task makes it possible to use Gitleaks within your Tekton pipelines.
+
+    Gitleaks is a SAST tool for detecting hardcoded secrets like passwords, api keys, and tokens in git repos.
+    It is an easy-to-use, all-in-one solution for finding secrets, past or present, in your code.
+  workspaces:
+    - name: source
+      description: A workspace that contains fetched git repo.
+  params:
+    - name: repo_path
+      type: string
+      description: path to the repo to be scanned.
+    - name: config_file_url
+      type: string
+      description: url from where the config file would be fetched.
+      default: ""
+    - name: config_file_path
+      type: string
+      description: path to config file.
+      default: ""
+    - name: output_format
+      type: string
+      description: output_format to use. (JSON|CSV|SARIF) (default:"json")
```

Comment:
this is a bit confusing.
shall we keep all the options and default in the same case